### PR TITLE
Feature/f 35 pdf preview implementation

### PR DESCRIPTION
**Pull request by bohdangarchu**
@Tschonti please approve if it is ok that I add `"react/jsx-no-bind": "off"` to our linter config. I think we need it for defining functions inside components like `handleScroll` (see `PdfViewer.tsx`).

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* PdfViewer.css */
+canvas.react-pdf__Page__canvas {
```
Comment (bohdangarchu) [inline src/styles/globals.css:6]: we need to use css here to add padding between pages

**Review by marinovl7 — CHANGES_REQUESTED** [review]
Overall great job! Take a look at my comments, just smal code styling issues I found. Also few things to add here: 

1. Can you please add the download PDF Functionality?
2. Can you think how we can get the documents from this endpoint `
https://static.hungermapdata.org/insight-reports/latest/country.json` and display them in the previewer? 
3. add border to the navbar and the color could be like light gray in light mode and dark grey in dark mode.

But overall quality and functionality is great! Good job!

File: tailwind.config.js [hunk 2]
```diff
@@ -26,7 +28,7 @@ module.exports = {
         light: {
           colors: {
             primary: { DEFAULT: '#157DBC', foreground: '#F5F5F5' },
-            secondary: '#EEEEEE',
+            secondary: '#666666',
```
Comment (bohdangarchu) [inline tailwind.config.js:31]: this corresponds to our secondary color used in figma designs

**Review by marinovl7 — APPROVED** [review]
Great Job!

**Pull request merged**